schemas.py

```diff
@@ -0,0 +1,46 @@
+from datetime import date
+
+from pydantic import BaseModel
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str
+    publication_date: date
+
+
+class BookCreate(BookBase):
+    author_id: int
+
+
+class Book(BookBase):
+    id: int
+
+    @classmethod
+    def from_orm(cls, obj):
+        return cls.from_attributes(**obj.__dict__)
+
```

Comment:
(https://docs.pydantic.dev/latest/migration/#changes-to-config)

The following config settings have been renamed:

orm_mode → from_attributes